content/en/docs/guides/runbooks/sharding-spinnaker.md

```diff
@@ -2,16 +2,16 @@
 title: "Sharding Spinnaker"
 linkTitle: "Sharding Spinnaker"
 weight: 2
-description: 
+description: "This document shows you how to shard traffic to different areas of Spinnaker"
```

Comment:
- Missing period at the end.
- I suggest adding a short "why" statement in the description too. Someone might come across the page and not know why they would want to shard traffic.

Reply:
I was thinking about that but I wasn't sure what to put exactly. I think it would be to scale the deployment to handle more traffic. I couldn't really find good documentation on this and I'm not familiar enough with Spinnaker to know the exact reasons why.

Reply:
I added a sentence on when sharding should be done.